lib/collection/script.js

```diff
@@ -12,6 +12,12 @@ var _ = require('../util').lodash,
  * @typedef {Object.<string, { id: string }>} Packages
  */
 
+/**
+ * A map of IDs to the names, location and other metadata and of the requests they refer to
+ *
+ * @typedef {Object.<string, { id: string, name: string }>} Requests
```

Comment:
can update the payload to reflect we are most likely gonna store:
- location
- workspace
- collection